.github/workflows/main.yaml

```diff
@@ -40,6 +40,20 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+      - name: Cache CI Essentials
+        uses: actions/upload-artifact@v4
+        with:
+          name: ci-essentials-${{ inputs.platform }}
+          include-hidden-files: true
+          retention-days: 1
```

Comment:
Change this to like 1 week just to be sure

Reply:
Done